### PR TITLE
build python 3.10

### PR DESCRIPTION
**Pull request by akaszynski**
Build, but don't test, Python 3.10. Testing unavailable due to requirement for < 9.1.0 on xvfb.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -24,33 +19,8 @@ def is_clang(bin):
     return not re.search(r'clang', output) is None
 
 
-def check_cython():
```
Comment (jgd10) [inline setup.py:27]: Is it OK to delete this?

File: requirements_test.txt [hunk 2]
```diff
@@ -3,6 +3,6 @@ pytest
 matplotlib
 pytest
 pytest-cov
-vtk<9.1.0
+vtk<9.1.0;python_version<"3.10"
```
Comment (jgd10) [inline requirements_test.txt:6]: So no VTK for Python 3.10?

**Review by jgd10 — APPROVED** [review]
LGTM! A couple of minor points/questions about it, but these are more for my information!

File: requirements_test.txt (hunk 2 above)
Comment (akaszynski) [reply requirements_test.txt:6]: There isn’t anything official at the moment. There are pyvista wheels compiled for 3.10 and a discussion about it here:
https://github.com/pyvista/pyvista/discussions/2064

File: setup.py (hunk 1 above)
Comment (akaszynski) [reply setup.py:27]: Yes, because we now check build requirements with pyproject.toml

**Pull request merged**